src/content/docs/style-guide/word-choice/user-related-language-guidelines.mdx

```diff
@@ -79,7 +79,7 @@ Example use:
  </td>
 
       <td>
-For docs site, use sentence case and bold. Alternatively, for non-docs-site styling, you can use italics or code-formatting: as long as it's clear that we're referring to something that has a definite, exact name. The words 'admin' and 'user' are generic, general words, so using formatting to indicate that we're referencing specific terms is important. 
+For docs site, use sentence case and bold. Alternatively, for non-docs-site styling, you can use italics or code-formatting: as long as it's clear that we're referring to something that has a definite, exact name. The words 'admin' and 'user' are generic, general words, so it can be important to format these to indicate that we're referencing specific terms. 
```

Comment:
same comment as before. Also, not sure why single quote around 'admin' and 'user'. Shouldn't those be backticks to make them code? Or bold? Or ""?